packaging/create_zipapp.py

```diff
@@ -15,6 +16,12 @@
 
 options = parser.parse_args(sys.argv[1:])
 
+try:
+    subprocess.check_call(options.interpreter + ' -c "import ctypes"', shell=True)
+except subprocess.CalledProcessError:
```

Comment:
Why is this not doing something like this instead:

```python
try:
    import ctypes
except ImportError:
   sys.exit('Python interpreter is lacking the ctypes module.')
```

Reply:
You may be targeting a different interpreter than the one doing the zipapp packaging